src/api/repo.ts

```diff
@@ -2,6 +2,94 @@ import { HYPERTRONS_OSS_XLAB_ENDPOINT, OSS_XLAB_ENDPOINT } from '../constant';
 import request, { mockSuccessRes } from '../utils/request';
 import { repoCorrelationData, developersByRepo } from '../mock/repo.data';
 
+// metric names and their implementation names in OpenDigger
+const metricNameMap = new Map([
+  ['activity', 'activity'],
+  ['openrank', 'openrank'],
+  ['participant', 'participants'],
+  ['forks', 'technical_fork'],
+  ['stars', 'stars'],
+  ['issues_opened', 'issues_new'],
+  ['issues_closed', 'issues_closed'],
+  ['issue_comments', 'issue_comments'],
+  ['PR_opened', 'change_requests'],
+  ['PR_merged', 'change_requests_accepted'],
+  ['PR_reviews', 'change_requests_reviews'],
+  ['merged_code_addition', 'code_change_lines_add'],
+  ['merged_code_deletion', 'code_change_lines_remove'],
+  ['merged_code_sum', 'code_change_lines_sum'],
+]);
+
+const getMetricByName = async (repo: string, metric: string) => {
+  const res = await request(
+    `${OSS_XLAB_ENDPOINT}/open_digger/github/${repo}/${metricNameMap.get(
+      metric
+    )}.json`
+  );
+  return res.data;
+};
+
+export const getActivity = async (repo: string) => {
+  return await getMetricByName(repo, 'activity');
```

Comment:
In Node.js, I think `return await Promise` is equal to `return Promise`, am I right? The behavior maybe the same for outside caller.

Reply:
You are right, thanks for pointing out that!